Launcher/src/windows/utils.vala

```diff
@@ -15,4 +15,10 @@ namespace Launcher {
 	string getBaseDirectory() {
 		return Environment.get_variable("APPDATA") + "\\.conquer";
 	}
+	
+	void tryUpdating() {
+		update(getBaseDirectory());
```

Comment:
The installation directory changed. The default plugins (DefaultMusic, ChangeCitiesMinds)/scenarios(Belenos, Etiona,...) are all installed in %PROGRAMFILES%/Conquer. (Together with the launcher, the dlls, the jars and uninstaller).

The location in %APPDATA% is only for user-specific content: Settings, saves, custom scenarios/plugins/...